src/controllers/verifyCommand.ts

```diff
@@ -1,8 +1,8 @@
 import config from "../../config/config";
+import { RETRY_COMMAND } from "../constants/responses";
 import { env } from "../typeDefinitions/default.types";
-import { discordTextResponse } from "../utils/discordResponse";
+import { discordephemeralResponse } from "../utils/discordEphemeralResponse.ts";
```

Comment:
Nitpik: make e capital `discordEphemeralResponse`

Reply:
Done, thanks for pointing out!